platform/mellanox/mlnx-onie-fw-update.sh

```diff
@@ -186,7 +186,12 @@ case "${cmd}" in
             rc=$?
             disable_onie_access
             if [[ ${rc} -eq 0 ]]; then
-                system_reboot
+                if [[ "${arg}" == "--no-reboot" ]]; then
+                    echo "ONIE firmware update successfully STAGED for install at NEXT reboot. Please reboot manually to complete installation."
+		    exit 0
```

Comment:
@alexrallen a typo?

Reply:
I don't see a typo, can you point it out?

Reply:
@alexrallen `exit 0` should be aligned with `echo ...`?

Reply:
Oh I see, think thats a tab / space thing in my editor will fix.